shared/devices/device-page/index.js

```diff
@@ -76,9 +77,15 @@ const Render = (props: Props) => {
     metaOne = <Meta title="revoked" style={styles.meta} backgroundColor={globalColors.red} />
   }
 
+  const icon: IconType = {
+    backup: 'icon-paper-key-64',
+    desktop: 'icon-computer-64',
+    mobile: 'icon-phone-64',
```

Comment:
Are there no 48 versions of these like the revoke ones?

Reply:
in zeplin its 64 on both